cli/command/formatter/cluster_health.go

```diff
@@ -102,71 +100,59 @@ func (c *clusterHealthContext) MarshalJSON() ([]byte, error) {
 
 func (c *clusterHealthContext) Node() string {
 	c.AddHeader(clusterHealthNodeHeader)
-	return c.v.Name
-}
-
-func (c *clusterHealthContext) Address() string {
-	c.AddHeader(clusterHealthAddressHeader)
-	u, err := url.Parse(c.v.AdvertiseAddress)
-	if err == nil && u.Host != "" {
-		return u.Host
-	}
-	return c.v.AdvertiseAddress
+	return c.v.NodeName
 }
 
 func (c *clusterHealthContext) healthy() bool {
 	return c.cpHealthy() && c.dpHealthy()
 }
 
 func (c *clusterHealthContext) cpHealthy() bool {
-	return c.v.Health.CP.NATS.Status+
-		c.v.Health.CP.KV.Status+
-		c.v.Health.CP.KVWrite.Status == "alivealivealive"
+	return c.v.Submodules.KV.Status+
+		c.v.Submodules.KVWrite.Status+
+		c.v.Submodules.NATS.Status == "alivealivealive"
 }
 
 func (c *clusterHealthContext) cpDegraded() string {
 	degraded := []string{}
-	if c.v.Health.CP.KV.Status != "alive" {
+	if c.v.Submodules.KV.Status != "alive" {
 		degraded = append(degraded, clusterHealthKVHeader)
 	}
-	if c.v.Health.CP.KVWrite.Status != "alive" {
+	if c.v.Submodules.KVWrite.Status != "alive" {
 		degraded = append(degraded, clusterHealthKVWriteHeader)
 	}
-	if c.v.Health.CP.NATS.Status != "alive" {
+	if c.v.Submodules.NATS.Status != "alive" {
 		degraded = append(degraded, clusterHealthNATSHeader)
 	}
 	return "Degraded (" + strings.Join(degraded, ", ") + ")"
 }
 
 func (c *clusterHealthContext) dpHealthy() bool {
-	return c.v.Health.DP.DirectFSClient.Status+
-		c.v.Health.DP.Director.Status+
-		c.v.Health.DP.FSDriver.Status+
-		c.v.Health.DP.FS.Status == "alivealivealivealive"
+	return c.v.Submodules.DirectFSInitiator.Status+
+		c.v.Submodules.Director.Status+
+		c.v.Submodules.Presentation.Status+
+		c.v.Submodules.RDB.Status == "alivealivealivealive"
```

Comment:
I know you've not changed them but err... why `alivealivealivealive`?  and how does that match up when looking for `alive` in `dpDegraded()`?